Add signing for fee estimation docs (#729)
software-mansion/starknet.py: docs/guide/signing.rst

.. codesnippet:: ../../starknet_py/tests/e2e/docs/guide/test_sign_offchain_message.py
    :language: python
    :dedent: 4

Signing for fee estimation
--------------------------

> :ref:`Account` allows signing transactions only for the purpose of fee estimation.

Reviewer: ```suggestion
:ref:`Account` allows signing transactions for the purpose of fee estimation.
```
With `only` it sounds to me like it's the only functionality.

> These transactions use a version that cannot be executed on StarkNet.

Reviewer: ```suggestion
These transactions use a version that makes them non-executable on StarkNet.
```
